Source/Fuse.Controls.Panels/TreeRenderer.iOS.uno

```diff
@@ -86,7 +86,11 @@ namespace Fuse.Controls
 
 		void ITreeRenderer.Placed(Element e)
 		{
-			_elements[e].SetSize(e.ActualSize);
+			var viewHandle = _elements[e];
+			if (viewHandle.NeedsRenderBounds)
+				viewHandle.SetSizeAndVisualBounds(e.ActualSize, e.RenderBoundsWithoutEffects);
```

Comment:
I recall observing renderbounds with both positive and negative values for position. And it looks correct in the Surface testapp